### PR TITLE
Add relative position to base picker

### LINKED ISSUE #26404: BasePicker should be position relative so that absolute child is always within bounds
In some cases, the absolute positioned accessible text can render outside of the bounds of picker causing overflow issues.

image

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #26404

Without relative position there is a chance that the absolute position child will cause layout issues.

### CODE REVIEW

File: change/@fluentui-react-4b696ee3-1985-435c-a365-c961bf362032.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-4b696ee3-1985-435c-a365-c961bf362032.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26490&commitId=be3645b5eece285a6f370139fa672eb2facc09c1&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|159| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4e8e21970639bc322699e1eb5b6832afe6493e65'>4e8e21970639bc322699e1eb5b6832afe6493e65</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount_base.html">2383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount.html">3062</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/BaseButton-mount_base.html">1529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/BaseButton-mount.html">1537</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Breadcrumb-mount_base.html">3609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Breadcrumb-mount.html">3613</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Checkbox-mount_base.html">3363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Checkbox-mount.html">3345</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/CheckboxBase-mount_base.html">3009</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/CheckboxBase-mount.html">3069</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ChoiceGroup-mount_base.html">5811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ChoiceGroup-mount.html">5844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ComboBox-mount_base.html">1571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ComboBox-mount.html">1550</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/CommandBar-mount_base.html">11841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/CommandBar-mount.html">11930</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ContextualMenu-mount_base.html">13597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ContextualMenu-mount.html">13683</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DefaultButton-mount_base.html">1812</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DefaultButton-mount.html">1768</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRow-mount_base.html">4598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRow-mount.html">4599</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRowFast-mount_base.html">4714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRowFast-mount.html">4896</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRowNoStyles-mount_base.html">4437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DetailsRowNoStyles-mount.html">4431</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Dialog-mount_base.html">3885</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Dialog-mount.html">3757</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DocumentCardTitle-mount_base.html">717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/DocumentCardTitle-mount.html">725</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Dropdown-mount_base.html">4165</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Dropdown-mount.html">4155</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/FocusTrapZone-mount_base.html">2495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/FocusTrapZone-mount.html">2510</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/FocusZone-mount_base.html">2422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/FocusZone-mount.html">2503</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount_base.html">2383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount.html">3062</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-virtual-rerender_base.html">1399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-virtual-rerender.html">1468</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">2124</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedList-virtual-rerender-with-unmount.html">2167</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-mount_base.html">692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-mount.html">707</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-virtual-rerender_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-virtual-rerender.html">679</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">709</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/IconButton-mount_base.html">2671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/IconButton-mount.html">2465</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Label-mount_base.html">906</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Label-mount.html">887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Layer-mount_base.html">5508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Layer-mount.html">5557</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Link-mount_base.html">1049</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Link-mount.html">1068</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/MenuButton-mount_base.html">2180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/MenuButton-mount.html">2185</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/MessageBar-mount_base.html">2887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/MessageBar-mount.html">2895</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Nav-mount_base.html">4151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Nav-mount.html">4197</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/OverflowSet-mount_base.html">1712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/OverflowSet-mount.html">1773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Panel-mount_base.html">3168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Panel-mount.html">3180</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Persona-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Persona-mount.html">1740</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Pivot-mount_base.html">2017</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Pivot-mount.html">2084</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/PrimaryButton-mount_base.html">1983</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/PrimaryButton-mount.html">1972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Rating-mount_base.html">9387</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Rating-mount.html">9361</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SearchBox-mount_base.html">2008</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SearchBox-mount.html">2009</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Shimmer-mount_base.html">3681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Shimmer-mount.html">3714</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Slider-mount_base.html">2756</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Slider-mount.html">2755</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SpinButton-mount_base.html">5931</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SpinButton-mount.html">6037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Spinner-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Spinner-mount.html">996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SplitButton-mount_base.html">4002</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SplitButton-mount.html">3934</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Stack-mount_base.html">1031</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Stack-mount.html">1057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/StackWithIntrinsicChildren-mount_base.html">3183</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/StackWithIntrinsicChildren-mount.html">3146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/StackWithTextChildren-mount_base.html">6374</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/StackWithTextChildren-mount.html">6417</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SwatchColorPicker-mount_base.html">13144</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/SwatchColorPicker-mount.html">13256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TagPicker-mount_base.html">3469</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TagPicker-mount.html">3363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TeachingBubble-mount_base.html">106635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TeachingBubble-mount.html">106334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Text-mount_base.html">992</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Text-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TextField-mount_base.html">2022</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/TextField-mount.html">2030</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-mount_base.html">1888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-mount.html">1878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-virtual-rerender_base.html">1326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-virtual-rerender.html">1321</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2672</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2702</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Toggle-mount_base.html">1389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/Toggle-mount.html">1359</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/buttonNative-mount_base.html">676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26490/perf-test/buttonNative-mount.html">662</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by spmonahan — APPROVED** [review]

**Review by smhigley — APPROVED** [review]

File: change/@fluentui-react-4b696ee3-1985-435c-a365-c961bf362032.json (hunk 1 above)
Comment (micahgodbolt) [reply change/@fluentui-react-4b696ee3-1985-435c-a365-c961bf362032.json:1]: unrelated

**Pull request merged**